Rakefile

```diff
@@ -165,6 +165,10 @@ namespace :plugin do
     rm_f package_output
     unity "-projectpath", project_path, "-exportpackage", "Assets", package_output
   end
+
+  task maze_runner: %w[plugin:export] do
+    sh "bundle", "exec", "bugsnag-maze-runner", "--verbose"
```

Comment:
`--verbose` has added a surprising amount of time onto the build times for other platforms, when we've finished debugging the CI setup it would be worth removing this.

Reply:
yeh the build is failing so I'm trying to find out why